api/src/wfl/module/xx.clj

```diff
@@ -2,196 +2,177 @@
-                        [in-bucket in-object] (gcs/parse-gs-url in-gs-url)]
-                    (when (and (= in-bucket bucket)
-                               (str/starts-with? object in-object))
-                      unsuffixed)))))]
-      (->> {:label cromwell-label
-            :status ["On Hold" "Running" "Submitted"]}
-           (cromwell/query environment)
-           (keep active?)
-           set))))
-
-(defn hold-workflow
-  "Hold OBJECT from IN-BUCKET for reprocessing into OUT-GS in
-  ENVIRONMENT."
-  [environment in-bucket out-gs object]
-  (let [path  (wdl/hack-unpack-resources-hack (:top workflow-wdl))
-        in-gs (gcs/gs-url in-bucket object)]
-    (cromwell/hold-workflow
-     environment
-     (io/file (:dir path) (path ".wdl"))
-     (io/file (:dir path) (path ".zip"))
-     (make-inputs environment out-gs in-gs)
-     (util/make-options environment)
-     cromwell-label-map)
-    (prn in-gs)))
-
-(defn on-hold-some-workflows
-  "Submit 'On Hold' up to MAX workflows from IN-GS to OUT-GS
-  in ENVIRONMENT."
-  [environment max-string in-gs out-gs]
-  (let [max (util/is-non-negative! max-string)
-        bucket (all/readable! in-gs)]
-    (letfn [(input? [{:keys [name]}]
-              (let [[_ unsuffixed suffix] (all/bam-or-cram? name)]
-                (when unsuffixed [unsuffixed suffix])))]
-      (let [done   (set/union (all/processed-crams (all/slashify out-gs))
-                              (active-objects environment in-gs))
-            suffix (->> in-gs
-                        gcs/parse-gs-url
-                        (apply gcs/list-objects)
-                        (keep input?)
-                        (into {}))
-            more   (->> suffix
-                        keys
-                        (remove done)
-                        (take max)
-                        (map (fn [base] (str base (suffix base)))))]
-        (run! (partial hold-workflow environment bucket out-gs) more)))))
-
-(defn release-some-on-hold-workflows
-  "Submit up to MAX-STRING workflows from 'On Hold' in ENVIRONMENT."
-  [environment max-string]
-  (letfn [(release! [id] (prn (cromwell/release-hold environment id)))]
-    (let [max (util/is-non-negative! max-string)]
-      (->> {:pagesize max :status "On Hold" :label cromwell-label}
-           (cromwell/query environment)
-           (map :id)
-           (take max)
-           (run! release!)))))
-
-(defn run
-  "Reprocess the BAM or CRAM files described by ARGS."
-  [& args]
-  (try
-    (let [env (wfl/throw-or-environment-keyword! (first args))]
-      (apply (case (count args)
-               4 on-hold-some-workflows
-               3 (partial all/report-status cromwell-label)
-               2 release-some-on-hold-workflows
-               (throw (IllegalArgumentException.
-                       "Must specify 2 to 4 arguments.")))
-             env (rest args)))
-    (catch Exception x
-      (log/error description)
-      (throw x))))
+(def references-defaults
```

Comment:
Some of this is defined in `references/hg38-exome-references`: https://github.com/broadinstitute/wfl/blob/master/api/src/wfl/references.clj#L41

Reply:
They're out of date with respect to the pipeline this module runs. I also want to be more explicit about what my defaults are. I want to see them in code, not a repl.

Reply:
Got it! Could we remove `references/hg38-exome-references` then? I think it would be confusing to leave them in if the exomes module will not use them

Reply:
nope - they're used in the `ukb` module.

Reply:
Oh I see!